python/sqlalchemy-openmldb/sqlalchemy_openmldb/saopenmldb.py

```diff
@@ -103,6 +103,9 @@ def dbapi(cls):
         from sqlalchemy_openmldb import openmldbapi as module
         return module
 
+    def has_table(self, connection, table_name, schema=None):
+        return table_name in connection.connection.cursor().get_all_tables()
```

Comment:
We do not support `schema` now. It would be better to check schema and throw exceptions if the `schema` is passed.

Reply:
updated.